src/AModule.cpp

```diff
@@ -3,6 +3,7 @@
 #include <fmt/format.h>
 
 #include <util/command.hpp>
+#include "sigc++/functors/mem_fun.h"
```

Comment:
That's weird, my editor did this automatically, but `mem_fun` was already called in this file and the include was not needed. Can revert if needed? I'm not that familiar with C++

Reply:
Yep remove this one :)